lang/en.json

```diff
@@ -225,6 +225,7 @@
     },
     "category": {
         "label": "Categories",
+        "product-label": "Product filter",
```

Comment:
@bc-vlad-dlogush for consistency, it is better to use an underscore in the naming of keys

Reply:
You are right, done!